subworkflows/nf-core/fasta_index_bismark_bwameth/main.nf

```diff
@@ -81,10 +83,10 @@ workflow FASTA_INDEX_BISMARK_BWAMETH {
     /*
     * Generate fasta index if not supplied for bwameth workflow or picard collecthsmetrics tool
     */
-    if (params.aligner == 'bwameth' | params.run_picard_collecthsmetrics) {
+    if (aligner == 'bwameth' | collecthsmetrics) {
```

Comment:
Is there a reason why this only works for bwameth?

Reply:
i ll let @dcarrillox answer that

Reply:
yes. because it needs the output of picard markdup which is run only in the bwameth workflow

Reply:
It is currently an "or", so presumably should be "and"?